shared/actions/config/index.js

```diff
@@ -75,6 +76,8 @@ function isFollowing(getState: () => any, username: string): boolean {
 
 const waitForKBFS = (): AsyncAction => dispatch =>
   new Promise((resolve, reject) => {
+    dispatch(fuseStatus())
```

Comment:
This is meant to initialize the state of fuse on startup (so in reducer state.favorite.fuseStatus is set), so we show the right banner on the folders tab. I could move it into the banner component mount instead?